### PR TITLE
add more owners.yaml files

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
I'd like to have a secondary owner for these sub directories since the single owner's are also usually the ones sending PR's for reviews

this is also a bug that i currently don't handle in the owner project code, if an OWNER file only has a single entry and also the Pull Request author it should not be eligible and should traverse up for a reviewer list.

**Comment by erwinmombay** [comment]
@cramforce do you mean [status checks](https://developer.github.com/v3/repos/statuses/) to block a PR from being merge-able? currently working on that

**Comment by cramforce** [comment]
@erwinmombay that is what I meant. You will need a little logo :)

**Comment by erwinmombay** [comment]
@cramforce the main issue im running into is that the "approvals" endpoint or data is not yet supported on any api (i contacted github support even). I originally was going to do a check where "has any of the owners in the list approved this PR", but since that isn't supported I would need to look for an LGTM comment or something

**Comment by cramforce** [comment]
Yes, lets do LGTM. I actually prefer that, because OWNERS may not have
rights to "Approve" changes.

File: extensions/amp-carousel/OWNER.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+- camelburrito
```
Comment (erwinmombay) [inline extensions/amp-carousel/OWNER.yaml]: @camelburrito can you suggest another owner for amp-carousel

File: extensions/amp-analytics/OWNER.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+- avimehta
```
Comment (erwinmombay) [inline extensions/amp-analytics/OWNER.yaml]: @avimehta can you suggest another owner for amp-analytics

Comment (avimehta) [inline extensions/amp-analytics/OWNER.yaml]: dima is most familiar with the code after me.

File: extensions/amp-carousel/OWNER.yaml (hunk 1 above)
Comment (erwinmombay) [inline extensions/amp-carousel/OWNER.yaml]: done

File: extensions/amp-analytics/OWNER.yaml (hunk 2 above)
Comment (erwinmombay) [inline extensions/amp-analytics/OWNER.yaml]: done

**Comment by erwinmombay** [comment]
wat.

**Comment by erwinmombay** [comment]
ah nevermind, my local env testing triggered an api call.

File: extensions/amp-form/OWNER.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+- mkhatib
```
Comment (jridgewell) [inline extensions/amp-form/OWNER.yaml]: 😭

**Comment by erwinmombay** [comment]
ok now thats gotta be a bug. review comment triggered a re-eval

**Comment by erwinmombay** [comment]
filed bug here https://github.com/google/github-owners-bot/issues/24

**Pull request merged**